README.md

```diff
@@ -1,9 +1,16 @@
 # Scylla monitoring with Grafana and Prometheus
 
-### Notice for users of Scylla version prior to 1.4
+<table style="border-style: solid;">
+  <tr>
```

Comment:
I've tried all options, and this one looks the best, have you seen how it looks compares to the other alternative?